coralproject/talk
# Routes and Layout Containers

## What does this PR do?

- Adds an AppRouter with Route nesting
- Abstracts into a LayoutContainer, and a Layout (UI) so we have a consistent layout wrapper
- Breaks Header and Drawer into UI componentes etc


## How do I test this PR?
- Run `npm run start` and `npm run build`
- Test that every route works as expected in `http://localhost:3000`


@coralproject/tech
